v4/weaviate/graphql/rawgql_test.go

```diff
@@ -0,0 +1,30 @@
+package graphql
+
+import "testing"
+
+func TestRawGQLBuilder_build(t *testing.T) {
```

Comment:
In addition to this test there was also a journey test which now disappeared... can you add back this journey test also?